lib/akismet.rb

```diff
@@ -24,6 +24,8 @@ def self.build_client
     end
 
     def comment_check(body)
+      return 'spam' if Rails.env.test? && body[:comment_content]&.include?('akismet-guaranteed-spam')
```

Comment:
Why do we need this hack for test?

Reply:
We're stubbing the Akismet request anyway, so it looks like we don't?